configs/relengworker-prod/config.yml

```diff
@@ -8,7 +8,35 @@ worker_types:
     autoscale:
       algorithm: slo
       args:
-        max_replicas: 20
+        max_replicas: 3
```

Comment:
We only have 12 beetmover jobs in a normal application-services release (e.g. https://firefox-ci-tc.services.mozilla.com/tasks/groups/IiPerdPzQ16jfNHgkLUYXQ) No need to ramp-up 20 workers, since the avg task time is less than 60 seconds and time is not a huge constraint. Let's decrease this to `3` instead.